### PR TITLE
feat(frontend): add Onramper ID for tokens

### PR DESCRIPTION
**Pull request by AntonioVentilii-DFINITY**
# Motivation

Another step to onboard Onramper is to map the correspondent Crypto ID to each token based [on this list](https://docs.onramper.com/docs/crypto-asset-support).

For now, we will set it just for ICP and ETH, but, after having tested more, map the rest.

# Changes

- Create new type to define the mapping to Crypto ID.
- Adjust RequiredToken type to excepts new type TokenBuyable.
- Set values for ICP and ETH.

### CODE REVIEW

File: src/frontend/src/lib/types/onramper.ts [hunk 1]
```diff
@@ -1,3 +1,7 @@
 // The list of networks that are supported by OnRamper can be found here:
 // https://docs.onramper.com/docs/network-support
 export type OnRamperNetworkId = 'icp' | 'bitcoin' | 'ethereum';
+
+// The list of cryptocurrencies that are supported by Onramper can be found here:
+// https://docs.onramper.com/docs/crypto-asset-support
+export type OnramperCryptoId = string;
```
Comment (peterpeterparker) [inline src/frontend/src/lib/types/onramper.ts:7]: What about just `OnramperId` given that's the term used in the linked documentation?

**Review by peterpeterparker — APPROVED** [review]
LGTM, thx

File: src/frontend/src/lib/types/onramper.ts (hunk 1 above)
Comment (AntonioVentilii-DFINITY) [reply src/frontend/src/lib/types/onramper.ts:7]: We will have a `OnRamperFiatId` too, I wanted to distinguish, even if they are both strings in the end, WDYT?

Comment (peterpeterparker) [reply src/frontend/src/lib/types/onramper.ts:7]: Honestly? When I open the link provided in the comment, https://docs.onramper.com/docs/crypto-asset-support, I started looking for the keyword "crypto" to find the ID

Comment (AntonioVentilii-DFINITY) [reply src/frontend/src/lib/types/onramper.ts]: ah! yes.. make sense, changed!

**Review by peterpeterparker — APPROVED** [review]
LGTM, thx

**Pull request merged**